terraform/ec2/linux/main.tf

```diff
@@ -57,18 +57,11 @@ resource "null_resource" "mount_efs" {
     host = aws_instance.cwagent.public_ip
   }
 
-  provisioner "file" {
-    source = "install-efs-utils.sh"
-    destination = "/tmp/install-efs-utils.sh"
-  }
-
   provisioner "remote-exec" {
     # https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-mount-helper-ec2-linux.html
     inline = [
-      "chmod +x /tmp/install-efs-utils.sh",
```

Comment:
@williazz will this work?

Reply:
Yes. It will work (double check with integration test)

Reply:
not sure why that works but okay

Reply:
after offline call with khan, i see that we moved the install-efs-utils script to the AMI building pipelines